tests/test_apply_imputation_link.py

```diff
@@ -0,0 +1,37 @@
+from pathlib import Path
+
+import pytest
+from helper_functions import load_and_format
+from pandas.testing import assert_frame_equal
+
+from src.apply_imputation_link import create_and_merge_imputation_values
+
+
+@pytest.fixture(scope="class")
+def fir_bir_c_fic_test_data():
+    return load_and_format(
+        Path("tests") / "data" / "apply_imputation_link" / "FIR_BIR_C_FIC.csv"
```

Comment:
Path("tests") / "data" / "apply_imputation_link" / "FIR_BIR_C_FIC.csv"
Not a big fan of this aproach, we combine windows path object with some strings

We can define the full path like this:
Path("tests","data","apply_imputation_link","FIR_BIR_C_FIC.csv")